EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -15237,7 +15237,7 @@ QUEST_LV_0100_20160718_015236	Ah well, I suppose it can't be helped.{nl}Could I
 QUEST_LV_0100_20160718_015237	There are some precious corals growing near the Kranta Skerries.{nl}Please provide the Fedimian Accessory Merchant Joana with the blue coral for the goods.{nl}

```

Comment:
`Please provide the Fedimian Accessory Merchant Joana with the blue coral for the goods`

to

`Please provide Joana, the Accessory Merchant of Fedimian with the blue coral for the goods`